### PR TITLE
Created template for documentation

### PR DESCRIPTION
**Pull request by Raghav888**

### CODE REVIEW

File: documentation/document.css [hunk 1]
```diff
@@ -1,150 +1,185 @@
...
+        display: flex;
+    }
+    /* Side bar */
+    
+    .side-bar {
+        width: 25%;
+        display: flex;
+        flex-direction: column;
+        height: max-content;
+        height: 100%;
```
Comment (ttahm3d) [inline documentation/document.css:63]: There are three values for the same property `height`

Comment (Raghav888) [reply documentation/document.css:63]: Will update it, Thank you

Comment (RohitCoder23) [reply documentation/document.css:63]: Yes, Make those changed and don't raise such big PR.
Why you named one html file as a.html. Give proper naming
 Rest looks good.

Comment (Raghav888) [reply documentation/document.css:63]: Yes Rohit will  make sure, not to repeat it.

**Pull request merged**